docs/current/742989-cookbook.md

```diff
@@ -173,6 +173,85 @@ The following code listing builds a single image for different CPU architectures
 
 [Learn more](./guides/406009-multiplatform-support.md)
 
+### Build image from Dockerfile
+
+The following code listing builds an image using an existing Dockerfile.
+
+<Tabs groupId="language">
+<TabItem value="Go">
+
+```go file=./quickstart/snippets/build-dockerfile/main.go
+```
+
+</TabItem>
+<TabItem value="Node.js">
+
+```javascript file=./quickstart/snippets/build-dockerfile/index.mjs
+```
+
+</TabItem>
+<TabItem value="Python">
+
+```python file=./quickstart/snippets/build-dockerfile/main.py
+```
+
+</TabItem>
+</Tabs>
+
+[Learn more](./quickstart/429462-quickstart-build-dockerfile.mdx)
+
+### Add OCI annotations to image
```

Comment:
I have noticed that in this recipe we use a local registry. Should we use `ttl.sh` instead, same as in the recipe prior to this one?

Alternatively, we should link in this section to instructions on how to set up a local registry. Maybe another recipe?

Reply:
Switched to ttl.sh